make-release.sh

```diff
@@ -68,12 +68,25 @@ if [[ ${NOCOMMIT} -eq 0 ]]; then
 fi
 
 if [[ $TRIGGER_RELEASE -eq 1 ]]; then
-  # push new branch to release branch to trigger CI build
+  # push new branch to release branch
   git fetch origin "${BRANCH}:${BRANCH}"
   git checkout "${BRANCH}"
   git branch release -f 
   git push origin release -f
```

Comment:
if this is a workflow dispatch (rather than on push to release branch) ... do we need this step?